### ISSUE TITLE
Sort not working on OSX

### ISSUE THREAD

**Issue by kaikuchn** [okbob/pspg#101]
Hey,

thanks for this awesome pager! This enables me to finally read wide result sets with ease thanks to horizontal scrolling and the navigation features.

I tried to use the sorting feature, but it does not work as expected. It does something but it isn't the right thing, see the attached gif. 

![sort](https://user-images.githubusercontent.com/1019068/64610645-03af7700-d3d0-11e9-91cc-dba4b3ec3c15.gif)

I also could not get the vertical cursor to show with `alt + v` while clicking the header or using the options menu does work.

I've installed pspg via homebrew:

```
❯ brew info pspg

pspg: stable 2.0.1 (bottled), HEAD
Unix pager optimized for psql
https://github.com/okbob/pspg
/usr/local/Cellar/pspg/2.0.1 (6 files, 139KB) *
  Poured from bottle on 2019-09-10 at 10:20:03
```

If there's any further info you need, please don't hesitate to ask. :)

Best regards,
Kai

**Comment by okbob** [comment]
Hi

sort is working only on numeric columns.

text are ignored - there are some unwanted artefact coming  from using
qsort over array where all fields are zero.

Pavel


út 10. 9. 2019 v 13:39 odesílatel Kai Kuchenbecker <notifications@github.com>
napsal:

**Comment by okbob** [comment]
I released 2.0.2 where sort on text columns is supported. Please, check it, test it.

https://github.com/okbob/pspg/releases/tag/2.0.2
